fpdf/graphics_state.py

```diff
@@ -56,6 +56,7 @@ def __init__(self, *args, **kwargs):
                 nom_lift=0.2,
                 denom_lift=0.0,
                 text_shaping=None,
+                font_size_set_for_page=False,
```

Comment:
I do not think that this attribute needs to be part of teh GraphicsState:
could you please try to remove it from this class?